core/test/fixtures/fraggle-rock/artifacts/step0/artifacts.json

```diff
@@ -163,12 +163,12 @@
     "throttlingMethod": "simulate",
     "screenEmulation": {
       "mobile": true,
-      "width": 360,
```

Comment:
This is probably good for this PR, but we should do another complete artifact refresh to capture any changes the screen adjustment had.